src/components/AddItemForm.js

```diff
@@ -43,7 +49,16 @@ class AddItemForm extends React.Component {
     });
   }
 
+  normalizeUserInput(item) {
```

Comment:
This is not a blocker in any way, and there are probably different schools of thought about this, but I think the `normalizeUserInput` method definition should be moved up to just after the `handleChange` method definition. That way a reader doesn't have to look around for `normalizeUserInput` when they see that `handleChange` calls it.

Reply:
I like this thought. Anything to help the readability of the code is good!